### PR TITLE
Review changes and update suggested code

### PR DESCRIPTION
**Pull request by vikaskbm**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 1]
```diff
@@ -1,11 +1,61 @@
...
+    value: "value5",
+  },
+];
+
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
   assigned_contact: null,
-  tag: null,
+  tags: null,
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:58]: If `tags` is an array, then its initial value should be `[]`, that is an `empty` array.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 2]
```diff
@@ -15,17 +65,18 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
...
-  tag: yup
-    .object()
+  tags: yup
+    .array(
+      yup.object().shape({
+        label: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.label)),
+        value: yup.string().oneOf(TAG_OPTIONS.map(tag => tag.value)),
+      })
+    )
     .nullable()
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:79]: Refer above comment and update this validation schema accordingly.

**Review by yedhink — CHANGES_REQUESTED** [review]
@vikas-bishnoi

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 1 above)
Comment (vikaskbm) [reply app/javascript/src/components/Dashboard/Notes/constants.js:58]: This is resolved.

PR #17

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 2 above)
Comment (vikaskbm) [reply app/javascript/src/components/Dashboard/Notes/constants.js:79]: This is resolved in #18

**Pull request merged**